site/docs/v1/tech/apis/scim/_scim-enterpriseuser-request-body.adoc

```diff
@@ -0,0 +1,72 @@
+Because SCIM schemas are completely customizable, There are no required fields for the incoming request other than the common fields that all SCIM requests must include to match the specification. The only requirement about the user data is that in the incoming request lambda, you must specify either the **email** or the **username** or both for the FusionAuth User object. Either of these values may be used to uniquely identify the User and may be used to authenticate the User. Otherwise, it is completely up to your defined lambda to match any other incoming user data to the appropriate FusionAuth User fields for your configuration.
```

Comment:
```suggestion
Because SCIM schemas are completely customizable, there are no required fields for the incoming request other than the common fields that all SCIM requests must include to match the specification.
The only requirement about the user data is that in the incoming request lambda, you must specify either the **email**, the **username**, or both for the FusionAuth User object.
Either of these values may uniquely identify the User and can be used during a login.
For all other fiends, it is completely up to your lambda to map incoming user data to the appropriate FusionAuth User fields.
```

Reply:
If you have multiple lines of text with no newlines between them, Asciidoc collapses them to a paragraph. Multiple lines are nice for diffs.